ethstore/src/bin/ethstore.rs

```diff
@@ -116,10 +116,13 @@ fn main() {
 
 fn key_dir(location: &str) -> Result<Box<KeyDirectory>, Error> {
 	let dir: Box<KeyDirectory> = match location {
-		"parity" => Box::new(ParityDirectory::create(DirectoryType::Main)?),
-		"parity-test" => Box::new(ParityDirectory::create(DirectoryType::Testnet)?),
-		"geth" => Box::new(GethDirectory::create(DirectoryType::Main)?),
-		"geth-test" => Box::new(GethDirectory::create(DirectoryType::Testnet)?),
+		"geth" => Box::new(RootDiskDirectory::create(paths::geth(false))?),
+		"geth-test" => Box::new(RootDiskDirectory::create(paths::geth(true))?),
+		path if path.starts_with("parity") => {
+			let chain = path.split('-').nth(1).unwrap_or("ethereum");
+			let path = paths::parity(chain);
+			Box::new(RootDiskDirectory::create(path)?)
```

Comment:
It works for `parity` (loads mainnet), `parity-testnet` is not really well defined currently, we can alias it to `parity-kovan` if you think it would be better.